scalalib/src/IvyDepsTreeArgs.scala

```diff
@@ -0,0 +1,44 @@
+package mill
+package scalalib
```

Comment:
```suggestion
package mill.scalalib
```

Can you place that file in the `mill.scalalib` subdir? I know the source directories are currently a bit of a mess. I'm going to clean this up later.

Reply:
Sure I'll move this.